packages/server/test/scripts/run.js

```diff
@@ -110,11 +110,14 @@ commandAndArguments.args.push(
   '--timeout',
   options['inspect-brk'] ? '40000000' : '10000',
   '--recursive',
-  '--compilers ts:@packages/ts/register,coffee:@packages/coffee/register',
+  '-r @packages/ts/register',
+  '-r @packages/coffee/register',
```

Comment:
this change breaks the server tests and makes any tests ending in `.ts` or `.coffee` not run. https://circleci.com/gh/cypress-io/cypress/297804?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link seems to be missing `newlines_spec.ts` output and `spec_spec.coffee` output, as well as many others...

when you revert this, can you add a comment noting NOT to use -r because it does not actually work? i feel like otherwise this change could easily crop up again

Reply:
@Bkucera still appears to not be running the vast majority of unit tests